go/tasks/plugins/webapi/bigquery/plugin.go

```diff
@@ -273,13 +280,54 @@ func (p Plugin) Status(_ context.Context, tCtx webapi.StatusContext) (phase core
 				resource.Status.ErrorResult.Message,
 				taskInfo), nil
 		}
-
+		err = writeOutput(ctx, tCtx, resource.OutputLocation)
```

Comment:
i'm confused, were we not writing an output before?

Reply:
We didn't write any output before.